dataland-frontend/src/utils/updateObjectUtils.ts

```diff
@@ -0,0 +1,45 @@
+export type objectType = { [key: string]: string | object };
+/**
```

Comment:
util typescript files should start with a capital letter in our project.

Please also rename `validationsUtils.ts" and `smoothScroll.ts` accordingly

Reply:
ok